src/main/java/org/sagebionetworks/web/client/widget/entity/renderer/VideoWidget.java

```diff
@@ -33,12 +34,19 @@ public void configure(final WikiPageKey wikiKey, final Map<String, String> widge
 		String webmSynapseId = descriptor.get(WidgetConstants.VIDEO_WIDGET_WEBM_SYNAPSE_ID_KEY);
 		String width = descriptor.get(WidgetConstants.VIDEO_WIDGET_WIDTH_KEY);
 		String height = descriptor.get(WidgetConstants.HEIGHT_KEY);
-		view.configure(wikiKey,	mp4SynapseId, oggSynapseId, webmSynapseId, width, height, authenticationController.isLoggedIn(), wikiVersionInView,
-				authenticationController.getCurrentXsrfToken());
+		view.configure(mp4SynapseId, oggSynapseId, webmSynapseId, width, height, authenticationController.getCurrentXsrfToken());
 		//set up view based on descriptor parameters
 		descriptor = widgetDescriptor;
 	}
 	
+	public void configure(String synapseId, String filename, int width, int height) {
```

Comment:
New configure method looks ok.  
The WidgetRendererPresenter.configure() is implemented if it is to be supported in the wiki markdown system.